docs/index.md

```diff
@@ -1,54 +1,17 @@
+You should maximize the probability that your party lives through an adventuring day. Page 84 of the Dungeon Master's Guide says most parties can handle 6-8 medium or hard encounters per day. The DMG also says the party will likely need to take two short rests during each of these days. At the end of the day, all of the decisions a party makes contribute to survival. The objective function needs to account for all of them (or as many as possible).
 
-Here, I propose the an optimization function: maximize the probability that your party lives through an adventuring day as defined by page 84 of the DMG.
-One such adventuring day is 6 medium difficulty encounters for a party of 4 PCs at level 5.
-The DMG suggests a short rest 1/3 and 2/3 the way through an adventuring day.
+Damage, control and defense are all measurable individually with the rules, but there isn't a way to compare how each one stacks up at the cost of the others. If you have great defenses and healing, then your enemies have a harder time killing the party. If you do great damage, then you get through your fights faster, and your enemies have a harder time killing the party. If you have great crowd control, then you can use conditions to delay or prevent enemy attacks, and your enemies have a harder time killing the party. The problem is not knowing which  gives your enemies the hardest time killing the party relative to the others. The goal of the objective function for making decisions should track the remaining resources at the end of an encounter.
 
-Why should that be the criteria? Why not optimize damage, or control, or utility, or some other measure?
-At the end of the day, all of those are *generalized* by "living through the day".
-At the end of the day, everything is reset, and you move on to your next day.
+The remaining resources the function should track are stats that persist between combat: Hit Points, Spell Slots, Rages, etc. If `choice A` leaves you with more remaining resources than `choice B` after one encounter, then `choice A` is better than `choice B`. The easiest decisions to compare are the ones where all of the resource types stay constant except one, so the function will focus on maximizing one particular resource.
 
-If you do great damage, then awesome! You got through your fights faster, and **your enemies have a harder time killing the party.**
-
-If you have great crowd control, then awesome! You got through your fights with more control, and **your enemies have a harder time killing the party.**
-
-If you have high utility, then awesome! You were able to cleverly bypass combat, or swing the battlefield to your advantage, and **your enemies have a harder time killing the party.**
-
-If we have to give up utility for crowd control, how do we decide? If we have to give up crowd control for damage, how do we decide?
-
-**You decide by doing the thing that maximizes the chance that your party lives to fight another day!**
```

Comment:
What was wrong with the bolding? I thought it helped hammer home the point that what mappers is not dying